exporter/zipkin/zipkin.go

```diff
@@ -50,6 +54,36 @@ func NewExporter(reporter reporter.Reporter, localEndpoint *model.Endpoint) *Exp
 	}
 }
 
+type Options struct {
+	// EndpointHostPort is the host:port combination for the Zipkin service
+	// e.g. 192.168.0.2:8080
+	EndpointHostPort string
+
+	// ReporterURI is the HTTP endpoint to which spans will be sent
+	// e.g. http://localhost:9411/api/v2/spans
+	ReporterURI string
+
+	// ReporterOptions are optional Options
+	// that modify the zipkin http Reporter.
+	ReporterOptions []ziphttp.ReporterOption
+}
+
+func NewExporterWithOptions(o *Options) (*Exporter, error) {
```

Comment:
NewExporter?

Reply:
Roger that, I'll replace the old with this one entirely.

Reply:
if you do, then you should preserve the ability to pass a non-HTTP reporter